use-cases/schedule-appointment/medconnect-service/appointment_api_config.bal

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

Comment:
Licence is Apache 2.0. Let's add the correct licence to all the source files

Reply:
Fixed in https://github.com/wso2/solutions-healthcare-demos/pull/8/commits/1f5aaedc782d76068b76a5b7936c80a5eece4298